recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ requirements:
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
     - setuptools
+    - libgcc  # [linux]
```

Comment:
I had the impression that during one of the last meetings we decided to depend strictly on `gcc` or `llvm` exactly for those reasons.